doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -227,7 +227,45 @@ The following list summarizes the most important changes inherited from upstream
 
 * Bluetooth:
 
-.. TODO
+  * Controller:
```

Comment:
Should this section be named `Zephyr Controller` instead? 

The default controller in NCS is the SoftDevice Controller. The following sentence may be interpreted as if there were bugfixes in the SoftDevice Controller
> Various Advertising Extensions feature related bug fixes 

That is not the case. @thomasstenersen may have input here.